test/TestSubject/TextPrinter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TestSubject
+{
+    public readonly struct StringSpan
+    {
+        public readonly string Source;
+        public readonly int Length;
+        public readonly int Offset;
+
+        public StringSpan(string source, int offset, int length)
+        {
+            this.Source = source;
+            this.Length = length;
+            this.Offset = offset;
+        }
+
+    }
+
+    public class TextPrinter
```

Comment:
Rename this class `MyClassWithInModifier`. The goal is to hint the _intent_ of the test. 
The file should be renamed `MyClass.InModifier.cs` too.